python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -483,6 +537,7 @@ def sum_source_flux(_source, exp, min_size):
 
     def center_and_cut_images(self):
         """ After defining sources for cwfs cut snippet for cwfs analysis.
+        This is used to speed up the CWFS analysis code
```

Comment:
It is not exactly to speed up the CWFS but to provide a snapshot with the donut for it to analyze.